Session1/EFCoreAssignment/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿// See https://aka.ms/new-console-template for more information
+using EFCoreAssignment;
+using EFCoreAssignment.Entities;
+using Microsoft.EntityFrameworkCore;
+
+Console.WriteLine("Hello, World!");
+
+var connection = @"Server=SVPHLP00642\SQLEXPRESS;Database=EFCoreMasters.Session01;Trusted_Connection=True;MultipleActiveResultSets=True"; // TODO : Add your connection string here
+var optionsBuilder =
+    new DbContextOptionsBuilder
+           <AppDbContext>();
+optionsBuilder.UseSqlServer(connection);
+
+var options = optionsBuilder.Options;
+
+var dbContext = new AppDbContext(options);
+
+Filtering(dbContext);
+SingleOrDefault(dbContext);
+LoadingRelatedData_Manual(dbContext);
+LoadingRelatedData_ExplicitLoading(dbContext);
+LoadingRelatedData_EagerLoading(dbContext);
+
+static void Filtering(AppDbContext dbContext)
+{
+    // TODO : Filter by Product Name
+    dbContext.Products.Where(c => c.Name == "Dress").ToList();
+}
+
+static void SingleOrDefault(AppDbContext dbContext)
+{
+    // TODO : Select using SingleOrDefault by Product Id   
+    dbContext.Products.SingleOrDefault(c=>c.Id==1);
+}
+
+static void LoadingRelatedData_Manual(AppDbContext dbContext)
+{
+    // TODO : Load Product with related shop data manually
+    var books = dbContext.Products;
+    var shops = dbContext.Shops;
+    foreach (var book in books)

```

Comment:
sometimes we forget to change variable names when we are copy pasting codes... I sometimes do that hehe... but I just want to point this out for readability...